mw-webapp/src/component/editableText/EditableText.tsx

```diff
@@ -72,7 +73,7 @@ export const EditableText = <T extends string | number>(props: EditableTextProps
    * Check type of coming value and convert it to Number if need to use input with type "number"
    */
   const setValue = (value: string) => {
-    const number = Number(value) ?? 0;
+    const number = FormatterUtils.defaultFormatter(value);
```

Comment:
method defaultFormatter return number or string but const naming is number. I think we need to change naming or update method for return only number